### PR TITLE
Adjust NetworkSize metric installer

### PR DESCRIPTION
**Pull request by freimair**
followup to #3997, part of https://github.com/bisq-network/projects/issues/13

Tweak the networksize metric scraper script, the pricenode installer and amend the readme on how to use it.

<details><summary>notes to self:</summary>
cannot make the JVM fire up a jmx-service.

standin jar works
```
% jcmd 28970 PerfCounter.print
[snip]
java.rt.vmArgs="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.local.only=true -Dcom.sun.management.jmxremote.port=6969 -Dcom.sun.management.jmxremote.rmi.port=6969 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"
[snip]
sun.management.JMXConnectorServer.0.authenticate="false"
sun.management.JMXConnectorServer.0.remoteAddress="service:jmx:rmi:///jndi/rmi://workstation:6969/jmxrmi"
sun.management.JMXConnectorServer.0.ssl="false"
sun.management.JMXConnectorServer.0.sslNeedClientAuth="false"
sun.management.JMXConnectorServer.0.sslRegistry="false"
sun.management.JMXConnectorServer.address="service:jmx:rmi://127.0.0.1/stub/rO0ABXNyAC5qYXZheC5tYW5hZ2VtZW50LnJlbW90ZS5ybWkuUk1JU2VydmVySW1wbF9TdHViAAAAAAAAAAICAAB4cgAaamF2YS5ybWkuc2VydmVyLlJlbW90ZVN0dWLp/tzJi+FlGgIAAHhyABxqYXZhLnJtaS5zZXJ2ZXIuUmVtb3RlT2JqZWN002G0kQxhMx4DAAB4cHc5AAtVbmljYXN0UmVmMgAADjE5Mi4xNjguOTMuMTAyAACyuWmYcsfkIQJcg+K0sQAAAXB3S81MgAIAeA=="
sun.management.JMXConnectorServer.remote.enabled=0
```
[snip]

price node does not
```
root@bisqserver ~ # /usr/lib/jvm/openjdk-10.0.2/bin/jcmd 18831 PerfCounter.print
[snip]
java.rt.vmArgs="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.local.only=true -Dcom.sun.management.jmxremote.port=6969 -Dcom.sun.management.jmxremote.rmi.port=6969 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"
[snip]
```

I can exclude these from the list of reasons
- java version
- using it with a .jar
- running with an unprivileged user
- is caused by service hardening

still open:
- [x] the pricenode is fired up via .jar and not with the startup scripts (used for seednodes for example) see https://github.com/bisq-network/bisq/pull/3997#pullrequestreview-363446739
- there seems to be an embedded tomcat within the pricenode, can that be the reason? but why should it be?

I currently tend to just leave it. It takes too much efforts to find the cause. We did not have any JVM heap data on pricenodes until now, we might visit that again once it issues show up.

### CODE REVIEW

File: pricenode/README.md
Comment (cbeams) [inline pricenode/README.md:72]: These "obligations" should be listed at https://bisq.wiki/Pricenode_Operator#Duties. I've just created that page as a stub. I've asked all @bisq-network/pricenode-operators to watch that page at https://github.com/bisq-network/roles/issues/14#issuecomment-591912853, so changes you make there will have a reasonable chance of being seen by them (duty changes should be communicated and followed up on more explicitly in any case, but FYI the wiki page is there, and we should rally around it).

File: pricenode/README.md
Comment (cbeams) [inline pricenode/README.md]:
```suggestion
Follow the instruction given by the script and report your certificate to the pricenode group!
```

What does "group" mean here?

**Review by cbeams — CHANGES_REQUESTED** [review]

File: pricenode/README.md
Comment (freimair) [reply pricenode/README.md:72]: agreed. however, I do not have the authority to create directives. Creating directives can only be done by
- the project leader,
- the team leader of the team that is affected (the ops-team in this case), or
- the team leader leader

at the time of writing.

File: pricenode/README.md
Comment (freimair) [reply pricenode/README.md]:
```suggestion
Follow the instruction given by the script and report your certificate to the monitoring group via [github](https://github.com/orgs/bisq-network/teams/monitoring-operators) or keybase (https://keybase.io/team/bisq channel #monitoring)!
```

Comment (cbeams) [reply pricenode/README.md]: "... report your certificate to the [`@bisq-network/monitoring`](https://github.com/orgs/bisq-network/teams/monitoring-operators) team or via the [Keybase](https://keybase.io/team/bisq) `#monitoring` channel."

File: pricenode/README.md
Comment (cbeams) [reply pricenode/README.md:72]: Ok. By the powers vested in me as Admin team lead, I request that you transcribe what you've written here to the wiki, and as part of the overall delivery of task's parent project bisq-network/projects#13, announce that change to the `@bisq-network/pricenode-operators` team, and follow up that each operator actually complies. If you're not comfortable doing this, then please manage that @wiz does it in his role as ops team lead. The point is that changes like this are useless unless they're effectively communicated and followed up on. It doesn't mean you have to do the work yourself, but that you own that the work does get done. Updating a readme (or the wiki for that matter) on its own won't carry the day. Feel free to create an admin repo issue or issues about doing this documentation and communication work and request/suggest that @wiz be assigned.

Comment (cbeams) [reply pricenode/README.md:72]: Note, the above should not hold up the merging of this PR.

**Review by cbeams — CHANGES_REQUESTED** [review]
I've left just one minor suggestion as to wording and some notes on follow up process.

**Review by wiz — APPROVED** [review]
ACK
running on gztmprecgqjq64zh pricenode, you should be getting data

File: pricenode/README.md
Comment (wiz) [reply pricenode/README.md:72]: I've just updated the wiki to add running collectd as a "duty" with link to the README anchor that will exist after this PR is merged

**Review by cbeams — APPROVED** [review]
utACK

**Pull request merged**
